style.css

```diff
@@ -193,16 +280,230 @@ h3 {
   background-color: #1a2236;
 }
 
-/* Aboutme Section css */
+/* Works Desktop Section CSS */
+
+@media (min-width: 768px) {
+  .works {
+    grid-template-columns: 26.8% 19.8% 16.8% 19.8% 16.8%;
+    grid-template-rows: 32.2% 4.8% 10.6% 10.6% 10.6% 15.6% 15.6%;
+    grid-auto-flow: row;
+    gap: 0 0;
+    height: 1660px;
+  }
+
+  .works img {
+    display: none;
+  }
+
+  .buttonproject img {
+    display: block;
+  }
+
+  h3 {
+    font-weight: 600;
+    font-size: 30px;
+  }
+
+  .html {
+    width: 132px;
+  }
+
+  .ruby {
+    width: 169px;
+  }
+
+  .js {
+    width: 144px;
+  }
+
+  .html,
+  .ruby,
+  .js {
+    height: 40px;
+    font-size: 20px;
+    line-height: normal;
+  }
+
+  .buttonproject {
+    height: 51px;
+    width: 212px;
+  }
+
+  .projecttitle {
+    display: flex;
+    align-items: center;
+    background-image: url(img/imagedots.svg);
+    background-position: 100% 80%;
+    background-repeat: no-repeat;
+    background-size: 130px;
+    transform: rotate(-90deg);
+    font-size: 110px;
+    margin: 0;
+    grid-area: 1 / 1 / 2 / 2;
+  }
+
+  .projectdescription.a {
+    grid-area: 1 / 2 / 2 /6;
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-start;
+    background-image: url(img/project1.png);
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
+
+  .projectinfo1 {
+    display: flex;
+    flex-direction: column;
+    background-color: #1a2236;
+    width: 100%;
+    height: 60%;
+    align-items: flex-start;
+    justify-content: space-evenly;
+    padding-left: 6%;
+  }
```

Comment:
[OPTIONAL] The title, languages, and button should be shown after the mouse is over the project card. You can implement them as hovers to improve your UI design.

<img width="694" alt="image" src="https://user-images.githubusercontent.com/77190203/161174240-9f69ff7d-4739-4089-8802-b9c30b452055.png">

Reply:
Hi @medaminedev66,
Yeah, I already know that I have to add these animations but these are for the next activity that I have to submit to my dashboard.

Reply:
Yes, I understand. It's optional.